snakemake/sourcecache.py

```diff
@@ -349,7 +365,7 @@ def _open(self, path_or_uri, mode):
             import git
 
             return io.BytesIO(
-                git.Repo(path_or_uri.repo_path)
+                git.Repo(path_or_uri.repo_path.replace("file:/",""))
```

Comment:
Maybe rather have this replace as a method in the LocalGitFile class?

Reply:
refactored and removed